README.md

```diff
@@ -98,17 +98,25 @@ up a PR and add it to the list.
 ## Getting started
 
 ### Installing CRI-O
+
 To install `CRI-O`, you can follow our [installation guide](install.md).
 Alternatively, if you'd rather build `CRI-O` from source, checkout our [setup
 guide](install.md#build-and-install-cri-o-from-source).
 We also provide a way in building [static binaries of `CRI-O`](install.md#static-builds) via nix.
 Those binaries are available for every successfully built commit on our [Google Cloud Storage Bucket][bucket].
-This means that the latest commit can be downloaded via:
+This means that the latest commit can be installed via our continence script:
 
 [bucket]: https://console.cloud.google.com/storage/browser/k8s-conform-cri-o/artifacts
 
 ```shell
-> curl -f https://storage.googleapis.com/k8s-conform-cri-o/artifacts/crio-$(git ls-remote https://github.com/cri-o/cri-o master | cut -c1-9).tar.gz -o crio.tar.gz
+> curl https://raw.githubusercontent.com/cri-o/cri-o/master/get | bash
```

Comment:
The url mentioned in https://github.com/cri-o/cri-o/pull/4587#discussion_r578314427 is still valid right? Because that's what I am using in https://github.com/kubernetes/test-infra/pull/20898

Would it be a good idea to have a complete URL, say as an example only for `amd64`, mentioned here? Just for someone who might not want to pipe a script from web into bash or someone like me who end up using complete URL in a systemd service file.

Reply:
Yes, that still applies. I changed the README.md to mention that URL path, too. But we have to be aware that this could cause a race (CI not finished or failed) in comparison to the script which will always succeed.

Reply:
Oh, ok, then in my case, I will change the systemd service file to first download the script and execute it locally.

Reply:
Yeah, this is way more safe and decouples our CI from the consumer of the script.

Reply:
Thanks. I was wondering, if it's possible to keep the old url around for sometime. It would take me sometime to get this tested and added in test-infra. Until then the periodic jobs, https://prow.k8s.io/?job=*crio*, will keep failing :( 

Just checking if it's possible, hate to see the prow dashboard in red :)

Reply:
I really like this approach @saschagrunert! The old one was kinda fragile, so thanks!